elements/mlnx-ofed/install.d/91-install-ofed-repo

```diff
@@ -6,8 +6,7 @@ fi
 set -eu
 set -o pipefail
 
-[ -d /etc/yum.repos.d ] || exit -1
```

Comment:
It would be nicer if you could predicate this on DISTRO_NAME, which DIB defines as part of its standard environment.  See https://github.com/stackhpc/stackhpc-image-elements/blob/master/elements/nvidia-cuda/install.d/95-nvidia-cuda for example